.gitattributes

```diff
@@ -0,0 +1 @@
+package-lock.json linguist-generated merge=ours
```

Comment:
what kind of sorcery is that

Reply:
oh this is magic. I've added `uv.lock` also in the sdk pr.

`linguist-generated` indicated that the file is generated, so it keeps it collapsed by default in PR's.

`merge=ours` is a merge strategy override, tells git to favour `ours` during rebases/merges.

Reply:
linguist-generated sounds good, but doesn't ours/theirs get reversed in rebase vs merge? I generally do rebases and mess those up, sure it's not a shoot in the foot?

Reply:
i'll be honest, the ours/theirs confuses me to this day, but i was told this was the correct one for lock files years ago and i've never had an issue

Reply:
i'll be honest, the ours/theirs confuses me to this day, but i was told this was the correct one for lock files years ago and i've never had an issue with this approach